src/main/java/io/github/BTEPlotSystem/utils/Utils.java

```diff
@@ -35,10 +38,19 @@
 import org.bukkit.configuration.file.FileConfiguration;
 import org.bukkit.inventory.ItemStack;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
 import java.util.UUID;
 
 public class Utils {
 
+    private static FileConfiguration config = BTEPlotSystem.getPlugin().getConfig();
```

Comment:
File configuration shouldn't be static, otherwise the data won't be changed during a config reload.

Reply:
Fixed in commit 99f0c18